eahub/base/views.py

```diff
@@ -244,5 +245,38 @@ def form_valid(self, form):
         return redirect("/{0}/{1}".format(type, reportee.slug))
 
 
+class SendMessageView(FormView):
+    template_name = "eahub/message.html"
+    form_class = SendMessageForm
+
+    def recipient(self):
+        return self.get_recipient()
+    
+    def form_valid(self, form):
+        message = form.cleaned_data
+        type = self.get_type()
+        current_user = self.request.user
+
+        # adds the group name to the subject line on emails to groups for clarity
+        def addressee(self):
+            if type == "group":
+                return self.recipient().name
+            else:
+                return "you"
+      
+        subject = f"{current_user.profile.name} sent {addressee(self)} a message through the Effective Altruism hub."
+        send_mail(
+            subject,
+            message,
+            current_user.email,
+            self. get_recipient_email()
+        )
```

Comment:
keywargs can be used here instead of variables, eg:
```
        send_mail(
            subject=f"{current_user.profile.name} sent {addressee}",
            message=form.cleaned_data["your_message"],
            receiver=self.request.user,
            desitinations=[self.get_recipient_email()]
        )
```